content/blog/2024-01-30-flash-framework.Rmd

```diff
@@ -0,0 +1,63 @@
+</blockquote>
+    
+Interestingly, through our prior system, I had the opportunity to experience all three roles, from developing the outlier detection methods, implementing and deploying the methods over Delphi's data, and then analyzing the outputs of the methods. These three roles came with different constraints and priorities. These experiences set the foundation for devising an interdisciplinary approach. 
+
+<center>
+![**Fig 2.** FlaSH's Human-in-the-loop-Framework](/blog/2024-01-30-flash-framework/image2.png)</center>
+
+Combining the methods, engineering, and quality assurance perspectives for event detection from large-scale public health streams, we developed a human-in-the-loop approach to address some fundamental challenges in existing alerting systems. Namely, we rank Delphi's recently updated data by how much it warrants reviewer attention. Reviewers can then decide how many data points they want to investigate, which prioritizes a reviewer's time and does not expect them to review a certain number of alerts flagged via some arbitrary threshold. 
+
+Here's a quick illustration of why the ranking approach works. The most limited resource in this setting is an expert reviewer's time and attention. We want to show this reviewer as many potential events as possible so that they can interpret the data with their known context. 
+
+<center>
+![**Fig 3.** Illustration of Ranking vs. Alerts for large data streams](/blog/2024-01-30-flash-framework/image1.png)</center>
+
+Now imagine each data point per signal was given an outlier score, and there is an arbitrary threshold where scores higher than that threshold are considered outliers. If we develop granular enough rankings, we can directly compare outlier points between different signals without arbitrary thresholds. Then, our data reviewer can focus on data points in their order of importance and stop the review at any time. 
+
+Developing this framework required new methods, visualizations, and evaluations. Over the past few years, our goal as part of the [FlaSH team](https://delphi.cmu.edu/blog/2024/01/01/introducing-the-flash-group/) was to implement and iterate on this framework. We're excited to share insights from these perspectives as we've moved FlaSH from a sandboxed research project to a deployed part of Delphi's systems that helps our users. For the next post in this series, I'll describe some of the novel methods we use to rank events in large volumes of public health data. Then, Richa and Nolan will share their thoughts on moving these research prototypes into a production environment. Third, Tina and Catalina will describe the data visualization and quality assurance processes and considerations they encountered while facilitating event detection investigation. Beyond that, we'll share updates on how we work towards tightening the human-in-the-loop process!
```

Comment:
Insights from Roni: The link to the other blog post is broken (the title of the linked article changed since this one was drafted, plus we should be able to use a relative URI reference) and we should use people's full names (i don't know Catalina's).
```suggestion
Developing this framework required new methods, visualizations, and evaluations. Over the past few years, our goal as part of the [FlaSH team](/blog/2024/01/01/introducing-flash-part-1-meet-the-team/) was to implement and iterate on this framework. We're excited to share insights from these perspectives as we've moved FlaSH from a sandboxed research project to a deployed part of Delphi's systems that helps our users. For the next post in this series, I'll describe some of the novel methods we use to rank events in large volumes of public health data. Then, Richa Gadgil and Nolan Gormley will share their thoughts on moving these research prototypes into a production environment. Third, Tina Townes and Catalina ____ will describe the data visualization and quality assurance processes and considerations they encountered while facilitating event detection investigation. Beyond that, we'll share updates on how we work towards tightening the human-in-the-loop process!
```